L1Trigger/TrackFindingTracklet/test/AnalyzerTracklet.cc

```diff
@@ -0,0 +1,247 @@
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "SimTracker/TrackTriggerAssociation/interface/StubAssociation.h"
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
+#include <TProfile.h>
+#include <TH1F.h>
+#include <TEfficiency.h>
+
+#include <vector>
+#include <deque>
+#include <set>
+#include <cmath>
+#include <numeric>
+#include <sstream>
+
+using namespace std;
+using namespace edm;
+using namespace trackerDTC;
+using namespace trackerTFP;
+using namespace tt;
+
+namespace trackFindingTracklet {
+
+  /*! \class  trackFindingTracklet::AnalyzerTracklet
+   *  \brief  Class to analyze TTTracks found by tracklet
```

Comment:
Change comment "found by tracklet" to "found by tracklet pattern recognition"

Reply:
done